sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -46,6 +36,20 @@ abstract class QueryTest extends PlanTest {
   Locale.setDefault(Locale.US)
 
   /**
+   * Makes sure the answer matches the expected result
+   */
+  def checkString(expected: String, actual: String): Unit = {
+    if (expected != actual) {
+      fail(
+        "The actual string gives wrong result:\n\n" + sideBySide(
```

Comment:
it's kind of an overkill, the `toString` we are testing always return a single line, just `assert` is enough.